test/functional/ui/statusline_spec.lua

```diff
@@ -892,3 +892,104 @@ describe('default statusline', function()
     ]])
   end)
 end)
+
+describe('float statusline #test', function()
```

Comment:
```suggestion
describe("'statusline' in floatwin", function()
```

Reply:
I think it should be `status-line` not `statusline` ..